src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+class IsoscelesTrapezoid implements Figure {
```

Comment:
The class name 'IsoscelesTrapezoid' is appropriate as it is a noun and represents a specific type of shape. However, ensure that if there are any behaviors that the class should not be responsible for, they are instead delegated to other classes or interfaces in accordance with the single-responsibility principle.